### PR TITLE
UAA memory resizing.

### PR DESCRIPTION
**Pull request by andreas-kupries**
Ref: https://trello.com/c/DVNMaxd0/735-5-evaluate-sizing-tests-and-update-docs-helm-charts-for-13

### CODE REVIEW

File: container-host-files/etc/scf/config/role-manifest.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ instance_groups:
               tag: mysql-data
               size: 20
           shared-volumes: []
-          memory: 1900
+          memory: 1400
```
Comment (mook-as) [inline container-host-files/etc/scf/config/role-manifest.yml:35]: The data has a max of 456; I guess we're still tripling this? (The nearest round number is 1536)

File: container-host-files/etc/scf/config/role-manifest.yml [hunk 2]
```diff
@@ -118,7 +118,7 @@ instance_groups:
           capabilities: []
           persistent-volumes: []
           shared-volumes: []
-          memory: 2205
+          memory: 2100
```
Comment (mook-as) [inline container-host-files/etc/scf/config/role-manifest.yml:121]: Ah, we're tripling everything (677 * 3 = 2031); though in that case 2048 would be a rounder number

File: container-host-files/etc/scf/config/role-manifest.yml (hunk 1 above)
Comment (andreas-kupries) [reply container-host-files/etc/scf/config/role-manifest.yml:35]: Yes, with some rounding up

File: container-host-files/etc/scf/config/role-manifest.yml (hunk 2 above)
Comment (andreas-kupries) [reply container-host-files/etc/scf/config/role-manifest.yml:121]: Heh. Do you care enough to want me to change it ?

**Review by mook-as — APPROVED** [review]
These numbers are no worse than anything else we pull out of a hat~~

**Pull request merged**